tests/test_model_training.py

```diff
@@ -38,6 +38,31 @@ def test_model_training(shapes_image_size, shapes_sample, shapes_sample_config,
     assert score.shape == (BATCH_SIZE, len(label_ids))
     assert all(0 <= s and s <= 1 for s in score.ravel())
 
-    BACKUP_FILENAME = os.path.join(str(shapes_temp_dir), "checkpoints", "test_model.h5")
+    BACKUP_FILENAME = os.path.join(str(shapes_temp_dir), "checkpoints", "test_model_{:02d}.h5".format(NB_EPOCHS))
     model.save(BACKUP_FILENAME)
     assert os.path.isfile(BACKUP_FILENAME)
+
+def test_model_backup_loading(shapes_image_size, shapes_sample_config, shapes_temp_dir):
+    """Test the model checkpoint recovering
+    """
+    config = read_config(shapes_sample_config)
+    label_ids = [x['id'] for x in config['labels'] if x['is_evaluate']]
+
+    cnn = FeatureDetectionNetwork("test", image_size=shapes_image_size, nb_labels=len(label_ids))
+    model = Model(cnn.X, cnn.Y)
+    old_weights = model.get_weights()
+    checkpoint_path = os.path.join(str(shapes_temp_dir), "checkpoints")
+    if os.path.isdir(checkpoint_path):
+        checkpoints = os.listdir(checkpoint_path)
+        if len(checkpoints) > 0:
+            model_checkpoint = checkpoints[-1]
```

Comment:
It's different from the previous `max(checkpoint_path)` in the implemented function `add_training_arguments`.

Reply:
Yes write, this is a small forget. This is supposed to be `max(checkpoints)`.